www/src/components/homepage/homepage-logo-banner.js

```diff
@@ -86,20 +28,50 @@ const HomepageLogoBanner = () => {
   `)
 
   return (
-    <Section>
-      <Title>
+    <section
+      sx={{
+        borderBottom: t => `1px solid ${t.colors.ui.border}`,
```

Comment:
I think you can do:

```suggestion
        borderBottom: t => `1px solid ${t.colors.ui.border}`,
        borderBottom: 1,
        borderColor: `ui.border`,
```

Reply:
ha just pushed right after you merged — needed a `borderStyle: "solid"` to make that work